eval.go

```diff
@@ -549,6 +760,38 @@ var (
 	valueBoolFALSE = reflect.ValueOf(false)
 )
 
+func ParseIndexExpr(baseExpression reflect.Value, indexExpression reflect.Value, indexType reflect.Type) (reflect.Value, error) {
```

Comment:
Can you refactor this function in to private ? I think is better to export only the names meant to be used by the user.

Reply:
I'd also add it to the Runtime struct like the other expression parsing methods.